diff --git a/Components/SystemButtons.qml b/Components/SystemButtons.qml
index 8db8572..04ee9f6 100644
--- a/Components/SystemButtons.qml
+++ b/Components/SystemButtons.qml
@@ -27,6 +27,7 @@ RowLayout {
             icon.source: modelData ? Qt.resolvedUrl("../Assets/" + modelData[0] + ".svgz") : ""
             icon.height: 2 * Math.round((root.font.pointSize * 3) / 2)
             icon.width: 2 * Math.round((root.font.pointSize * 3) / 2)
+            icon.color: palette.text
             display: AbstractButton.TextUnderIcon
             visible: config.ForceHideSystemButtons != "true" && modelData[2]
             hoverEnabled: true
